### ISSUE TITLE
Deep import of abstract-control in angular material lib

### ISSUE THREAD

**Issue by JBBianchi** [eclipsesource/jsonforms#1784]
### Describe the bug

Not a bug but there is a deep import of [abstract-control](https://github.com/eclipsesource/jsonforms/blob/18ffe5e15672ce2dded35fd86dc742bee0b1ab2d/packages/angular/src/abstract-control.ts) in [array-layout](https://github.com/eclipsesource/jsonforms/blob/18ffe5e15672ce2dded35fd86dc742bee0b1ab2d/packages/angular-material/src/layouts/array-layout.renderer.ts#L32).

```
Warning: Entry point '@jsonforms/angular-material' contains deep imports into 'C:/Dev/Sandbox/jsonforms-angular-seed/node_modules/@jsonforms/angular/lib/abstract-control'. This is probably not a problem, but may cause the compilation of entry points to be out of order.
```

### Expected behavior

[abstract-control](https://github.com/eclipsesource/jsonforms/blob/18ffe5e15672ce2dded35fd86dc742bee0b1ab2d/packages/angular/src/abstract-control.ts) should be exported by the module ('s [index](https://github.com/eclipsesource/jsonforms/blob/18ffe5e15672ce2dded35fd86dc742bee0b1ab2d/packages/angular/src/index.ts)).

### Steps to reproduce the issue

install & build angular-seed

### In which browser are you experiencing the issue?

None

### Framework

Angular

### RendererSet

Material

**Comment by sdirix** [comment]
Thanks for the report!

**Comment by eneufeld** [comment]
This is the reason for #1781 and should be fixed with #1785
